eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -196,7 +201,24 @@ function Upload-Blobs
     Write-Host "DocVersion $($DocVersion)"
     Write-Host "DocDir $($DocDir)"
     Write-Host "Final Dest $($DocDest)/$($PkgName)/$($DocVersion)"
+    Write-Host "Release Tag $($ReleaseTag)"
 
+    # Use the step to replace master link to release tag link 
+    if ($ReleaseTag) {
+        foreach ($htmlFile in (Get-ChildItem $DocDir -include *.html -r)) 
+        {
+            $fileContent = Get-Content -Path $htmlFile
+            $checkPattern = $false
+            while ($fileContent -match $RepoReplaceRegex) {
+                $fileContent = $fileContent -replace $RepoReplaceRegex, "`${1}$ReleaseTag`$2"
+                $checkPattern = $true
+            }
+            if ($checkPattern) {
+                Set-Content -Path $htmlFile -Value $fileContent
+            }
+        }
```

Comment:
I would add the warning message here in the else clause if we don't have a release tag state that we aren't able to update the links to use the release tag because we weren't able to find one.